From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Mon, 13 Apr 2020 10:00:48 -0400
Subject: [PATCH] redhat: Replace hardware.redhat.com link in Unsupported
 message

Resending ... did not get any reviews.

P.

---8<---

Bugzilla: http://bugzilla.redhat.com/1810301
Brew: https://brewweb.engineering.redhat.com/brew/taskinfo?taskID=27902812
Local Tree: git://git.engineering.redhat.com/users/prarit/rhel8 1810301

Upstream Status: RHEL only

For almost a decade, RHEL kernels have output the message

"this hardware has not undergone testing by Red Hat and might not be
certified. Please consult https://hardware.redhat.com for certified
hardware."

for unsupported hardware.  This link should be now be updated to the
new catalog.redhat.com location.  The old hardware.redhat.com is now
redirecting to catalog.redhat.com for older releases.

Signed-off-by: Prarit Bhargava <prarit@redhat.com>
Cc: David Arcari <darcari@redhat.com>
---
 kernel/rh_taint.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kernel/rh_taint.c b/kernel/rh_taint.c
index 8d1641710ed7..4050b6dead75 100644
--- a/kernel/rh_taint.c
+++ b/kernel/rh_taint.c
@@ -24,7 +24,7 @@
 void mark_hardware_unsupported(const char *msg)
 {
 	/* Print one single message */
-	pr_crit("Warning: %s - this hardware has not undergone testing by Red Hat and might not be certified. Please consult https://hardware.redhat.com for certified hardware.\n", msg);
+	pr_crit("Warning: %s - this hardware has not undergone testing by Red Hat and might not be certified. Please consult https://catalog.redhat.com for certified hardware.\n", msg);
 }
 EXPORT_SYMBOL(mark_hardware_unsupported);

-- 
2.28.0

